#include <iostream>
#include <exception>

using namespace std;

/*

class HashTable***Exception: public exception
{

    virtual const char* what() const throw(){
	return "HashTable***Exception thrown";	
    }

};

*/
